### ISSUE TITLE
Dynamic props

### ISSUE THREAD

**Issue by iamfrntdv** [styled-components/vue-styled-components#23]
Props are not dynamic. If variable or state updates it's value styled-component won't be updated

vue file:
```vue
<template lang="pug">
	item(:active="active", @click.native='setActive()') Change My Color Please!
</template>

<script>
	import item from './item';

	export default {
		name: 'item',
		data () {
			return {
				active: false
			};
		},
		methods: {
			setActive () {
				this.active = true
			}
		}
	}
</script>
```

style-component:
```js
import styled from 'vue-styled-components';

const itemProps = { active: Boolean };

const item = styled('div', itemProps)`
	color: ${props => props.active ? 'red' : 'green'};
`

export default item;
```
This just won't work

**Comment by liqueflies** [comment]
HI @iamfrntdv,
I confirm this issue.

I'm running into this to solve ASAP.

Thank you.

**Comment by liqueflies** [comment]
@iamfrntdv happy to tell and share with you that issue was resolved!

Thanks for reporting me. 🎉

**Comment by iamfrntdv** [comment]
After latest update doesn't work... (

**Comment by liqueflies** [comment]
@iamfrntdv Checking right now.

**Comment by liqueflies** [comment]
Fixed in 1.2.1, was a merge mistake.

Thanks for reporting :)
